package.json

```diff
@@ -25,28 +26,53 @@
   "homepage": "https://github.com/vuejs/vue-cli#readme",
   "main": "lib/index.js",
   "scripts": {
-    "test": "npm run lint && npm run e2e",
-    "lint": "eslint test/e2e/test.js lib bin/* --env mocha",
-    "e2e": "rimraf test/e2e/mock-template-build/*.* && cross-env BABEL_ENV=development mocha test/e2e/test.js --slow 1000 --compilers js:babel-core/register"
+    "test": "npm run lint && npm run e2e && npm run e2e:build",
+    "lint": "eslint test/e2e/test*.js lib bin/* --env mocha",
+    "e2e": "rimraf test/e2e/mock-template-build && cross-env BABEL_ENV=development mocha test/e2e/test.js --slow 1000 --compilers js:babel-core/register",
+    "e2e:build": "cross-env BABEL_ENV=development mocha test/e2e/test-build.js --timeout 60000 --compilers js:babel-core/register"
   },
   "dependencies": {
     "async": "^2.0.0-rc.2",
+    "autoprefixer": "^6.6.1",
+    "buble-loader": "^0.4.0",
     "chalk": "^1.1.1",
     "commander": "^2.9.0",
+    "connect-history-api-fallback": "^1.3.0",
     "consolidate": "^0.14.0",
+    "css-loader": "^0.26.1",
     "download-git-repo": "^0.2.1",
+    "express": "^4.14.0",
+    "extract-text-webpack-plugin": "2.0.0-beta.4",
+    "file-loader": "^0.9.0",
+    "friendly-errors-webpack-plugin": "^1.1.2",
     "handlebars": "^4.0.5",
+    "html-webpack-plugin": "^2.26.0",
+    "http-proxy-middleware": "^0.17.3",
     "inquirer": "^0.12.0",
+    "installed-by-yarn-globally": "^0.1.1",
     "metalsmith": "^2.1.0",
     "minimatch": "^3.0.0",
     "multimatch": "^2.1.0",
+    "opn": "^4.0.2",
     "ora": "^0.2.1",
+    "post-compile-webpack-plugin": "^0.1.0",
+    "postcss-loader": "^1.2.1",
     "read-metadata": "^1.0.0",
     "request": "^2.67.0",
     "rimraf": "^2.5.0",
     "semver": "^5.1.0",
+    "tildify": "^1.2.0",
     "uid": "0.0.2",
-    "validate-npm-package-name": "^2.2.2"
+    "url-loader": "^0.5.7",
+    "user-home": "^2.0.0",
+    "validate-npm-package-name": "^2.2.2",
+    "vue": "2.1.8",
```

Comment:
Did you intentionally fix version to `2.1.8`? I'd say we want to give latest Vue 2 version whenever user installs or updates `vue-cli`. What do you think? (same for `vue-template-compiler` and maybe even `webpack` - I heard 2.2.0 non RC is coming soon).